docs/release-notes/release-notes-0-18-0.md

```diff
@@ -132,6 +132,11 @@ The following defects were fixed in this release.
 |   |   |
 |   |   |
 |   |   |
+| GUI-1331  | CSV option is not available for import and export button for Document or Graph Edge collections.  |
+|   |   |
+| GUI-1874  | For Scale tier accounts, instance size details are not displayed in the Account tab. |
```

Comment:
@diana-macrometa GUI-1874 this ticket is part self service provisioning so, we are thinking to remove this feature. We don't require to have it in 18.0.
cc: @VikasPatel  @rahulprabhune-macrometa

Reply:
Removed.